py-polars/polars/internals/frame.py

```diff
@@ -49,6 +49,7 @@
     sequence_to_pydf,
     series_to_pydf,
 )
+from polars.internals.lazy_functions import all as pyall
```

Comment:
It is imo better to import `polars.internals.lazy_functions.all` in `polars.internals.__init__`, and then in this file to refer to it using `pli.all`.  Otherwise the code becomes hard to reason about if we are to do `import X as Y` on every item we use.

Reply:
Changed this in https://github.com/pola-rs/polars/pull/2136/commits/49fe96bba13b3e9cc90ac78b7d3d58de8bb6df05